registry/handlers/app.go

```diff
@@ -377,6 +377,9 @@ func (app *App) RegisterHealthChecks(healthRegistries ...*health.Registry) {
 			if _, ok := err.(storagedriver.PathNotFoundError); ok {
 				err = nil // pass this through, backend is responding, but this path doesn't exist.
 			}
+			if err != nil {
+				dcontext.GetLogger(app).Error(err)
```

Comment:
Let's prefix this error with some context letting us know that this was from the storage driver health check, something like `fmt.Errorf("storage driver health check: %v", err)`. Otherwise, it might be a little hard to know what's happening looking at the logs without prior knowledge of this change.

Reply:
good idea - done ✔️